### DISCUSSION TITLE
Stractraces snippet (Markdown / Asciidoc)

### DISCUSSION THREAD

**Discussion by bric3** [community/community#27584] [Other Feature Feedback, Questions, & Ideas]
Have you thought about supporting stacktrace snippets.

Jetbrains Youtrack markup, has a specific support for stacktraces, the code block highlight some specific text, like `at` in a JVM stack, and is collapsible. See stacktrace section in https://youtrack.jetbrains.com/markup

````markdown
```stacktrace
java.lang.NullPointerException
        at com.ClassA.someMethod(ClassA.java:20)
        at com.ClassA.someMethod2(ClassA.java:10)

```
````

````asciidoc
[source,stacktrace]
----
java.lang.NullPointerException
        at com.ClassA.someMethod(ClassA.java:20)
        at com.ClassA.someMethod2(ClassA.java:10)

----
````

Currently Github renders this as text
> ```stacktrace
> java.lang.NullPointerException
>         at com.ClassA.someMethod(ClassA.java:20)
>         at com.ClassA.someMethod2(ClassA.java:10)
> 
> ```

**Comment by airtower-luna** [comment]
The format for stacktraces/stack tracebacks/whatever-they-are-called varies as heavily by programming language as the exact term used. I don't think it makes sense to have anything other than rendering them as text, and they're usually made to be readable as text. Unless you want to add separate highlighting for a lot of programming languages, like `java-stacktrace`, `py-traceback`, … :thinking:

**Reply by bric3** [reply]
Of course ! I forgot to mention that but indeed I had the idea of supporting multiple languages.
I don't mind having to specify the language as well in the identifier (like `stacktrace-java`).
